python/python-quiz.md

```diff
@@ -2191,6 +2191,14 @@ print("programming".center())
 - [x] Guido van Rossum
 - [ ] Alan Turing
 
+
+#### Q159. Which collection is ordered, changeable, and allows duplicate members?
```

Comment:
It brakes questions number sequence, may be it worth to add question as a last, and simply increment number.  
Or you could change numbers of all next questions after this one.